ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/MockCrypto.hs

```diff
@@ -66,8 +67,15 @@ type CanMock era =
   , Arbitrary (Core.PParams era)
   , Arbitrary (Core.Script era)
   , Arbitrary (Core.TxBody era)
+  , Arbitrary (Core.TxInBlock era)
   , Arbitrary (Core.TxOut era)
   , Arbitrary (Core.Value era)
   , Arbitrary (PredicateFailure (SL.UTXOW era))
-  , CL.TxSeq era ~ SL.TxSeq era
+  )
+
+-- | A stronger 'CanMock' that is incompatible with Alonzo
+type CanMockOld era =
```

Comment:
If this is only applicable to eras before Alonzo, I wouldn't call it `CanMockOld` but instead `CanMockPreAlonzo` or something along those lines.

Reply:
This was done. Closing.